src/test/java/net/sf/jabref/model/database/BibDatabaseTest.java

```diff
@@ -197,4 +197,124 @@ public void changingEntryPostsChangeEntryEvent() {
 
         assertEquals(entry, tel.getBibEntry());
     }
+
+    @Test
+    public void correctKeyCountOne() {
+        BibEntry entry = new BibEntry();
```

Comment:
Init the entry with key `AAA` in the `@Before` method so that you can reuse it in every test.